### ISSUE TITLE
screenshots the slup selector edges

### ISSUE THREAD

**Issue by Idiot64** [emersion/slurp#84]
looks weird, you see black edges on the screenshot, most likely becuase it also screenshots slurp

**Comment by emersion** [comment]
I can't reproduce.

**Comment by Idiot64** [comment]
hm, i can send you one 
![image](https://user-images.githubusercontent.com/70104078/116790240-7b557c80-aab3-11eb-9143-0fff98dcdc92.png)

also, this issue might be becuase i use wayfire, but I'm not sure

**Comment by videogame-hacker** [comment]
> might be because i use wayfire

Do you have some animation that fades windows out when they close instead of them instantly disappearing? Does this happen to slurp's border?

**Comment by MrSwedish** [comment]
ye, it does that, but couldn't it just ignore the box?

**Comment by emersion** [comment]
No, slurp nor grim can't do anything about Wayfire's animations.

**Comment by tmccombs** [comment]
perhaps wait a small amount of time between getting the coordinates with grim. Something like:

```
grim -g "$(slurp; sleep .5)" -| wl-copy
```

**Comment by lilydjwg** [comment]
> No, slurp nor grim can't do anything about Wayfire's animations.

slurp can do one thing: set an app_id so that it can be matched and have animation disabled. Currently I find no way to identify slurp so I have to disable all animation for the "overlay" type.

**Comment by emersion** [comment]
slurp sets the layer-shell namespace. layer-shell has no app-id.

**Comment by lilydjwg** [comment]
Oops, that explains. Thanks.
